util/dvsim/FlowCfg.py

```diff
@@ -18,6 +18,10 @@
 # A set of fields that can be overridden on the command line.
 _CMDLINE_FIELDS = {'tool'}
 
+# A set of fields to resolve as paths, relative to the hjson file that sets
```

Comment:
Hiding the detail of which hjson keys are 'relative' paths in the python code does not seems like a good approach to me. I would prefer to keep the existing approach (use {proj_root} or {dvsim_tool_root} as anchors to specify full paths to files).

Reply:
Ok, I've thrown out the "Read flow_makefile relative..." patch and changed the import in `common_sim_cfg.hjson` to use `dv_root`.